webrtc.html

```diff
@@ -1644,6 +1644,13 @@ <h4>Set the RTCSessionDescription</h4>
                             truncate <var>sendEncodings</var> so that its length is
                             <var>supportedEncodings</var>.
                           </li>
+                          <li>
+                            If <var>sendEncodings</var> is non-empty, set each
+                            encoding's
+                            {{RTCRtpEncodingParameters/scaleResolutionDownBy}}
+                            to <code>2^(<var>supportedEncodings</var> - encoding
+                            index - 1)</code>.
```

Comment:
We cannot use the maximum number of encodings supported by the client for the scaling factors. You need to use the number of encodings instead.
A client supporting 3 layers would have very different results from another supporting 6 layers for example.

Reply:
You're right I was thinking they were the same but that's only if truncation happens. Updated PR.